src/main/kotlin/uk/gov/dwp/dataworks/services/EcsDescribeServicesCall.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.dwp.dataworks.services
+
+import org.springframework.stereotype.Service
+import software.amazon.awssdk.services.ecs.EcsClient
+import software.amazon.awssdk.services.ecs.model.DescribeServicesRequest
+import software.amazon.awssdk.services.ecs.model.DescribeServicesResponse
+
+@Service
+class EcsDescribeServicesCall{
```

Comment:
I don't see the benefit of this being a distinct class (or even method)? Since it's a very small amount of code, used once, I don't seen any benefit to separating it like this

Reply:
I separated these out, in order to mock it in the tests. I couldn't work out another way to do it.

Reply:
You could separate it out to a method in the class, as opposed to a full class in itself and mock the method that way.